python/runtime/cudaq/algorithms/py_draw.cpp

```diff
@@ -39,6 +39,37 @@ std::string pyDraw(py::object &kernel, py::args args) {
 }
 
 /// @brief Bind the draw cudaq function
-void bindPyDraw(py::module &mod) { mod.def("draw", &pyDraw, ""); }
+void bindPyDraw(py::module &mod) {
+  mod.def(
+      "draw", &pyDraw,
+      R"#(Return a UTF-8 encoded string representing drawing of the execution 
+path, i.e., the trace, of the provided `kernel`.
+      
+Args:
+  kernel (:class:`Kernel`): The :class:`Kernel` to draw.
+  *arguments (Optional[Any]): The concrete values to evaluate the kernel 
+    function at. Leave empty if the kernel doesn't accept any arguments.
+
+Returns:
+  The UTF-8 encoded string of the circuit, without measurement operations.
+
+.. code-block:: python
+
+  # Example
+  import cudaq
+  @cudaq.kernel
+  def bell_pair():
+      q = cudaq.qvector(2)
+      h(q[0])
+      cx(q[0], q[1])
+      mz(q)
+  print(cudaq.draw(bell_pair))
```

Comment:
It would be good to have an example here in the docs that requires runtime arguments, so that users know draw requires them.

Reply:
Addressed in 16049a5